playbooks/archivematica-centos7/requirements.yml

```diff
@@ -4,14 +4,22 @@
   version: "2.x"
   name: "elastic.elasticsearch"
 
-- src: "https://github.com/jdauphant/ansible-role-nginx"
+- src: "https://github.com/artefactual-labs/ansible-percona"
   version: "master"
-  name: "jdauphant.nginx"
+  name: "artefactual.percona"
+
+- src: "https://github.com/artefactual-labs/ansible-nginx"
+  version: "master"
+  name: "artefactual.nginx"
 
 - src: "https://github.com/artefactual-labs/ansible-clamav"
   version: "master"
   name: "artefactual.clamav"
 
+- src: "https://github.com/artefactual-labs/ansible-gearman.git"
+  version: "dev/centos-support"
```

Comment:
We should merge dev/centos-support branch into master, instead of using custom branches